src/animated.tpp

```diff
@@ -130,7 +126,7 @@ inline void animated<T>::update_last_draw_time(double acceleration)
 	}
 
 	if(get_current_frame_end_time() < get_animation_time() && // catch up
-			get_current_frame_end_time() < get_end_time()) {  // don't go after the end
+		get_current_frame_end_time() < get_end_time()) {      // don't go after the end
```

Comment:
Can you put these two conditions on one line.

Reply:
done